portfolio/src/main/webapp/me.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
```

Comment:
https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags Maybe get rid of these extra tags?

Reply:
I'm confused by this. How would I be able to link my stylesheet and script then?

Reply:
hmmm... good point 🤔 I don't know, so maybe don't worry 'bout it lol